Makefile

```diff
@@ -1,7 +1,8 @@
 all: server/index.html server/cpkthttp
 
 server/index.html: client/index.html
-	./utils/deflate.py client/index.html server/index.html
+	sed 's:^\s*\|\s*$\|/\*.*\*/::g' client/index.html | tr -d '\n' > server/index.html.stripped
+	zopfli --i1000 -c --deflate server/index.html.stripped > server/index.html
```

Comment:
@evq will this run ok on the server?

Reply:
not atm, but that's easily fixed

Reply:
@diracdeltas this should be good to go server side now